packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -239,6 +240,17 @@ class PaginatedDataTable extends StatefulWidget {
   /// The index of the first row to display when the widget is first created.
   final int? initialFirstRowIndex;
 
+
+  /// {@template flutter.material.dataTable.dividerThickness}
+  /// The width of the divider that appears between [TableRow]s.
+  ///
+  /// Must be greater than or equal to zero.
+  /// {@endtemplate}
```

Comment:
These dartdoc templates exist so that we can type out a description in 1 place and reuse it.
Consider changing to:

```suggestion
  /// {@macro flutter.material.dataTable.dividerThickness}
```